src/components/Container/Container.scss

```diff
@@ -0,0 +1,117 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  text-align: center;
+  max-width: $breakpoint-laptop-large;
+  margin: 0 16px;
+  padding: 5px;
+}
+
+// This column class could be removed entirely and the grid system would work. We could consiser adding the padding and margin to the layout classes, and controlling this in the layout classes.
+
+.column {
+  padding: 10px;
+  margin: 10px;
+}
```

Comment:
I dont think we need this column logic see comment below

Reply:
@mattlaguardia .column class removed.